python/mage/union_find/__init__.py

```diff
@@ -0,0 +1,4 @@
+from mage.union_find.union_find import (  # noqa: F401, F402, F403
+    UnionFind,
+    Node,
+)
```

Comment:
Delete `python/__init__.py`. Couldn't make comment there since it is empty.

Reply:
Done!